### ISSUE TITLE
Build example for Adobe's UXP framework for Photoshop plugins

### ISSUE THREAD

**Issue by danbgoldman** [gradio-app/gradio#9718]
- [X] I have searched to see if a similar issue already exists.

**Is your feature request related to a problem? Please describe.**  
I attempted to build the JS client as part of a UXP project (Adobe's javascript runtime for Photoshop plugins). Unfortunately, I'm only a n00b at javascript builds - especially webpack - and I wasn't able to get too far. It's possible that the UXP environment doesn't support some functionality required for correct operation of the client, but it could just be a configuration problem on my part. Since there are many available gradio apps and HF spaces that perform image processing and image generation, it would be exciting to be able to build Photoshop plugins that use gradio APIs!

**Describe the solution you'd like**  
I'd like to see a build example for a simple UXP plugin that connects to an HF space using the client API. If it's not possible today in UXP, please work with Adobe to make it happen!

**Additional context**

**Comment by abidlabs** [comment]
Hi @danbgoldman we'll need a more self-contained example and repro for us to investigate this. Could you provide us more details about what you tried and what errors you ran into?

**Comment by danbgoldman** [comment]
I ended up changing to a different approach, so I don't have a simple example handy that I can easily reconstruct. In short, the [UXP environment](https://developer.adobe.com/photoshop/uxp/2022/uxp-api/reference-js/) only supports a subset of the standard node libs that seemed to be required by the gradio client. I attempted to set up a webpack environment that would import polyfills for these, but when trying to upload an image file from my plugin, I got an inscrutable [Object object] error. I have a hunch it's because UXP doesn't support SSE, but I didn't investigate far enough to be sure. I found there were other features in UXP that are documented but don't work properly, eg. ImageBlob.text() always returns an empty string. So this is most likely a bug for Adobe, not gradio.

If I have time I will try to construct a simple test case.
